docs/src/creating-packages.md

```diff
@@ -46,6 +49,20 @@ greet() = print("Hello World!")
 end # module
 ```
 
+The content of `test/Project.toml` is:
+```toml
+[deps]
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+```
+
+The content of `test/runtests.jl` is:
+```jl
+using HelloWorld
+using Test
+
+@test 1 == 1
```

Comment:
I would maybe propose the structure that PkgTemplates uses:

```jl
@testset "PkgName.jl" begin
    @test 1 == 1
end
```

Reply:
FWI, PkgDev used to generate a failing test by default.